cpp/src/svm/linear.cu

```diff
@@ -374,8 +376,10 @@ void fit(const raft::handle_t& handle,
                   2,
                   psi,
                   &target,
-                  &num_iters,
+                  &n_iter,
                   (T*)sampleWeight);
+
+    if (n_iter > max_n_iter) { max_n_iter = n_iter; }
```

Comment:
`qnFit` sets `n_iter` to 0 and increment from there. Wouldn't we want to output the sum of the first fit and the callibration?

Reply:
No. `max_iter` controls the maximum number of iterations per fit. Since `probability=True` does two fits (and uses the same `max_iter` both times) we want to record the maximum number of iterations used across all fits. Setting `max_iter=some_successful_fit.n_iter_` should work (barring any non-determinism, which I don't think we have here).

```
max_iter = LinearSVC().fit(X, y).n_iter_
LinearSVC(max_iter=max_iter).fit(X, y)
```